packages/react/src/contexts/treasure.tsx

```diff
@@ -227,6 +239,13 @@ const TreasureProviderInner = ({
     setUser(nextUser);
     setStoredAuthToken(nextAuthToken as string);
 
+    trackCustomEvent({
+      name: EVT_TREASURECONNECT_CONNECTED,
```

Comment:
Should this also go in the `logIn` function in the core package?

Reply:
Can add it there too — wasn't sure how much we wanted to track in tdk-core since not all of it's relevant